CMakeLists.txt

```diff
@@ -464,7 +464,7 @@ set(OFFSETS_O_PATH ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/offsets.dir/arch/${ARC
 set(OFFSETS_H_PATH ${PROJECT_BINARY_DIR}/include/generated/offsets.h)
 
 add_library(          offsets STATIC ${OFFSETS_C_PATH})
-target_link_libraries(offsets zephyr_interface)
+target_link_libraries(offsets zephyr_interface -u_OffsetAbsSyms -u_ConfigAbsSyms )
```

Comment:
This is out-of-scope so it doesn't have to block the PR, but if you wish you could instead of moving the -u_OffsetAbsSyms flag assist me in getting rid of it.

https://github.com/zephyrproject-rtos/zephyr/pull/8547